debouncer.go

```diff
@@ -60,6 +62,7 @@ func (d *Debouncer) Stop() {
 
 // Start the debouncer
 func (d *Debouncer) Start() {
+	d.timer = time.NewTicker(d.interval)
```

Comment:
I think we should clean the previous timer here if it exists but it also works for me

Reply:
I will the `time.NewTicker` in the `New` function, assuming that we can't Start a debouncer that was already Stopped